src/_components/_display/markdown/wrappedMarkdown.tsx

```diff
@@ -0,0 +1,23 @@
+import Markdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw';
+import remarkGfm from 'remark-gfm';
+import remarkToc from 'remark-toc';
+
+export type WrappedMarkdownProps = {
+  text: string;
+};
+
+export const WrappedMarkdown = ({ text }: WrappedMarkdownProps) => {
+  const cleanText = text
+    .replace(/<\/?script>/g, 'Naughty!')
```

Comment:
This seems quite permissive as sanitization - does it work with newline/null char injection [etc](https://cheatsheetseries.owasp.org/cheatsheets/XSS_Filter_Evasion_Cheat_Sheet.html)? Are there any other filter evasion techniques we could nix with this (I'm guessing we're limited in sanitization due to it being html content)

Reply:
I've done some more read-around on the component, and I can explicitly block tags and their content a lot more easily than I did here. For the most part it's not that big an issue - we're not allowing direct use of html in the editor for one - so this is more an emergency backstop in case someone does an API intercept.

Reply:
Is it worth just throwing a big old list of XSS attacks at the function and seeing what happens? There's a few example repos out there with various lists of attacks, e.g. [this one](https://github.com/payloadbox/xss-payload-list)